jetty-maven-plugin/pom.xml

```diff
@@ -13,15 +13,11 @@
     <bundle-symbolic-name>${project.groupId}.maven.plugin</bundle-symbolic-name>
     <jetty.stopKey>FREEBEER</jetty.stopKey>
     <jetty.jvmArgs></jetty.jvmArgs>
+    <jacoco.skip>true</jacoco.skip>
+    <maven.test.skip.exec>true</maven.test.skip.exec>
```

Comment:
Why do we have this for `jetty-maven-plugin` at all?
And we had a similar maven-surefire-plugin/config/skip=true on `jetty-jspc-maven-plugin` and you didn't include this `maven.test.skip.exec` property there.